app/src/main/java/org/akvo/flow/ui/adapter/ResponseListAdapter.java

```diff
@@ -77,11 +79,9 @@ public void bindView(View view, Context context, Cursor cursor) {
 
         String username = cursor.getString(SurveyDbAdapter.FormInstanceQuery.SUBMITTER);
         if (TextUtils.isEmpty(username)) {
-            userView.setVisibility(View.GONE);
-        } else {
-            userView.setVisibility(View.VISIBLE);
-            userView.setText(username);
+            username = DEFAULT_USERNAME;
```

Comment:
The view is `VISIBLE` by default then, right?

Reply:
yes, the view is visible by default